src/bitmessagekivy/tests/test_allmail_message.py

```diff
@@ -0,0 +1,41 @@
+from .telenium_process import TeleniumTestProcess
+from .common import skip_screen_checks
+from .common import ordered
+
+
+class AllMailMessage(TeleniumTestProcess):
+    """AllMail Screen Functionality Testing"""
+
+    @skip_screen_checks
+    @ordered
+    def test_show_allmail_list(self):
+        """Show All Messages on Mail Screen/Window"""
+        try:
```

Comment:
write a separate method in telenium process class something like `telenium_wait_for_launch`
```
start = time.time()
while not timeout:
    try:
       assertExists(selector)
       return ok
    except IOError:
       pass
    except assertError:
       failed to start
    sleep(0.1)
```